projects/control-service/projects/pipelines_control_service/src/test/resources/org/apache/tika/mime/custom-mimetypes.xml

```diff
@@ -0,0 +1,5 @@
+<mime-type type="application/pyc">
```

Comment:
The folder structure here is actually different from the one used in main. 
Here is it folders whereas in main it is a dot separated file name.
 
 Also given that this is the same as the one in main how do you actually know this is being picked up in tests? 
 I would just remove this if it is the same and then you are sure that the tests are testing what will be deployed.